uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/ScoreList.kt

```diff
@@ -61,6 +61,31 @@ class ScoreList {
         return result
     }
 
+    /**
+     * Returns the number of skips after the offset in the interval used to calculate
+     * the percentage of completed days.
+     *
+     * If skips are found in the interval, it is expanded until the interval has the size of the
+     * sum of the denominator and the number of skips within the interval.
```

Comment:
There's no `denominator` in this function so this doc could be improved to make it understandable on its own, can you try to do that?